### PR TITLE
Add gitignore and requirements

### PR DESCRIPTION
**Pull request by SiarheiZhachek**

### CODE REVIEW

**Comment by SiarheiZhachek** [comment]
ok

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+exceptiongroup==1.1.2
+iniconfig==2.0.0
+packaging==23.1
+pluggy==1.2.0
+pytest==7.4.0
+tomli==2.0.1
```
Comment (mikyyL) [inline requirements.txt:6]: test

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+exceptiongroup==1.1.2
+iniconfig==2.0.0
+packaging==23.1
+pluggy==1.2.0
+pytest==7.4.0
```
Comment (mikyyL) [inline requirements.txt:5]: pytest

**Review by mikyyL — COMMENTED** [review]
test

File: requirements.txt (hunk 1 above)
Comment (writeln2012) [reply requirements.txt:6]: +

**Review by writeln2012 — APPROVED** [review]

**Pull request merged**